plugins/modules/status.py

```diff
@@ -24,7 +24,7 @@
 
 EXAMPLES = r"""
 - name: Read status from pulp api server
-  status:
+  pulp.squeezer.status:
     pulp_url: localhost:24817
```

Comment:
unrelated, but I think you could unify the `pulp_url`s ;)

Reply:
Yes, i could.
https://github.com/pulp/squeezer/pull/98